data/events/scripts/player.lua

```diff
@@ -41,64 +41,52 @@ local storeItemID = {
 -- Players cannot throw items on teleports if set to true
 local blockTeleportTrashing = true
 
-local titles = {
-	{ storageID = 14960, title = " Scout" },
-	{ storageID = 14961, title = " Sentinel" },
-	{ storageID = 14962, title = " Steward" },
-	{ storageID = 14963, title = " Warden" },
-	{ storageID = 14964, title = " Squire" },
-	{ storageID = 14965, title = " Warrior" },
-	{ storageID = 14966, title = " Keeper" },
-	{ storageID = 14967, title = " Guardian" },
-	{ storageID = 14968, title = " Sage" },
-	{ storageID = 14969, title = " Tutor" },
-	{ storageID = 14970, title = " Senior Tutor" },
-	{ storageID = 14971, title = " King" },
-}
-
 local config = {
 	maxItemsPerSeconds = 1,
 	exhaustTime = 2000,
 }
 
-if not pushDelay then
-	pushDelay = { }
-end
+local pushDelay = { }
+
+local function antiPush(player, item, count, fromPosition, toPosition, fromCylinder, toCylinder)
```

Comment:
Consider moving this function to "player.lua" in the core folder or turning it into a Player:antiPush(...
and keep using the self